javascript/javascript2/week1/danishLetters.js

```diff
@@ -0,0 +1,46 @@
+
+function danishLetters(str) {
+  let letter_count1 = 0; // this "å" counter
+  let letter_count2 = 0; // this "ø" counter
+  let letter_count3 = 0; // this "æ" counter
+  
+  for (let i = 0; i < str.length; i++) {
```

Comment:
think of creating an javascript object instead `var danishLetter= {æ : 0, ø: 0, å:0};` where the **property** is the danish letter and the **value** of the key is the number of the danish letter appearances . See https://www.w3schools.com/js/js_objects.asp